app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -148,6 +151,17 @@ class User private constructor(
             this.description = description
         }
 
+        /**
+         * Set the age range in the UserBuilder
+         *
+         * @param ageRange a list containing the age range :
+         *     ageRange[0] = minAge,
+         *     ageRange[1] = maxAge
+         */
+        fun setAgeRange(ageRange: List<Int>) = apply {
+            this.ageRange = ageRange
+        }
+
```

Comment:
I think it may be useful to check for the size of the list before setting it to the user, so that we can avoid bugs with lists of more than 2 elements (and less than 2)